Pinta.Core/Effects/Utility.cs

```diff
@@ -433,7 +431,7 @@ public static double GetNearestStepAngle (double angle, int steps)
 		double fullTurn = 2 * Math.PI;
 		double stepAngle = fullTurn / steps;
 		double normalizedAngle = angle % fullTurn;
-		int sector = Convert.ToInt32 (Math.Truncate ((normalizedAngle % fullTurn) / stepAngle));
+		int sector = Convert.ToInt32 (Math.Truncate (normalizedAngle % fullTurn / stepAngle));
```

Comment:
Looking closer, I'm not sure why we need `normalizedAngle % fullTurn` when the previous line does `normalizedAngle = angle % fullTurn`

Reply:
Good point. I've now removed that part